src/app/Common/Services/Interfaces/ITokenService/Models/UserToken.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Common.Enums;
+using Common.Utils;
+
+namespace Common.Services.Interfaces.ITokenService
+{
+    public class UserToken : IExpirable
```

Comment:
I'd name it `UserTokenModel`, because it seems convenient to differentiate between classes, representing MongoDB objects (i.e. derived from BaseDocument - User, Token) and all other models. So we usually add `Model` (in some cases `ViewModel`, `DTO`, etc) to names of these classes.